rsciio/tests/test_tiff.py

```diff
@@ -37,23 +37,21 @@
 import rsciio.tiff
 
 
-MY_PATH = os.path.dirname(__file__)
-MY_PATH2 = os.path.join(MY_PATH, "tiff_files")
+TEST_DATA_PATH = Path(__file__).parent / "data" / "tiff"
+TEST_NPZ_DATA_PATH = Path(__file__).parent / "data" / "npz"
 TMP_DIR = tempfile.TemporaryDirectory()
 
 
 def teardown_module():
     TMP_DIR.cleanup()
```

Comment:
`TMP_DIR` still needed as `tmp_path` is now used?

Reply:
Yes, because with unzipping file it is not convenient to use the pytest fixture